xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/editors/dataeditors.css

```diff
@@ -20,7 +20,9 @@
 }
 /* Basic form elements styling */
 #xwikiobjects input, #xwikiobjects textarea,
-#xwikiclassproperties input, #xwikiclassproperties textarea {
+#xwikiobjects select, #xwikiobjects .selectize-input,
+#xwikiclassproperties input, #xwikiclassproperties textarea,
+#xwikiclassproperties select, #xwikiclassproperties .selectize-input {
   border: 1px solid $theme.borderColor;
   width: 90%;
```

Comment:
The correct solution here is to use .xform

Reply:
@evalica You mean replacing `#xwikiclassproperties` and `#xwikiobjects` with `.xform`?

Reply:
I don't think ``dataeditors.css`` should know about ``selectize.js`` classes.

Reply:
Makes sense, I'll move the related code in `selectize.css`.

Reply:
in 10.7 we apparently have the xform class added to form#update. So instead of adding more selectors, I would just remove the border and the width declarations and rely on .xform+Skin styling.
Before
<img width="1327" alt="before" src="https://user-images.githubusercontent.com/629552/43585977-1124986a-966f-11e8-8f45-fc87eeaa0633.png">

After
<img width="1313" alt="after" src="https://user-images.githubusercontent.com/629552/43585983-144ec2ae-966f-11e8-9218-f303e4c9f551.png">


for http://localhost:8080/xwiki/bin/edit/Help/Applications/Movies/Meet%20John%20Doe?editor=object&force=1 with 
```
#xwikiobjects input, #xwikiobjects textarea, #xwikiclassproperties input, #xwikiclassproperties textarea {
    /* border: 1px solid #E8E8E8; */
    /* width: 90%; */
}
```
in dataeditors.css

Reply:
With suggest objects type, After: 
<img width="1327" alt="suggest" src="https://user-images.githubusercontent.com/629552/43586148-83f6194a-966f-11e8-8cb8-9ef69a8ab862.png">